external/fv3fit/fv3fit/emulation/data/transforms.py

```diff
@@ -0,0 +1,108 @@
+"""
+Array transforms to build onto our batching pipeline into training.
+Generally assumes data to be in Sample x Feature shape.
+"""
+
+import logging
+import numpy as np
+import tensorflow as tf
+import xarray as xr
+from toolz.functoolz import curry
+from typing import Mapping, Sequence, Tuple, Union
+
+
+logger = logging.getLogger(__name__)
+
+NumericContainer = Union[np.ndarray, xr.DataArray, tf.Tensor]
+StandardizeInfo = Mapping[str, Tuple[NumericContainer, NumericContainer]]
+FeatureSubselection = Mapping[str, slice]
+ArrayDataset = Mapping[str, np.ndarray]
+TensorDataset = Mapping[str, tf.Tensor]
+InputDataset = Mapping[str, Union[np.ndarray, xr.DataArray]]
+AnyDataset = Mapping[str, NumericContainer]
+
+
+def to_ndarrays(dataset: xr.Dataset) -> ArrayDataset:
+    """Convert a dataset to ndarrays with a specified dtype."""
+    logger.debug("Converting dataset to ndarray dataset")
+    return {varname: da.values for varname, da in dataset.items()}
+
+
+@curry
+def to_tensors(dataset: InputDataset, dtype: tf.DType = tf.float32) -> TensorDataset:
+    """Convert a dataset to tensors with specified dtype."""
+    logger.debug("Converting dataset to tensor dataset")
+    return {
+        key: tf.convert_to_tensor(data, dtype=dtype) for key, data in dataset.items()
+    }
+
+
+@curry
+def select_antarctic(dataset: xr.Dataset, sample_dim_name="sample") -> xr.Dataset:
+    """
+    Select only points below 60 S.  Requires 'latitude' in dataset and expects
+    units in radians
```

Comment:
I believe the default for lats/lons is radians in fv3, but let me know if this is not the case.